src/main/java/de/adesso/anki/battle/world/Body.java

```diff
@@ -1,14 +1,20 @@
 package de.adesso.anki.battle.world;
 
+import java.util.List;
+
 import org.eclipse.paho.client.mqttv3.MqttException;
 
 import de.adesso.anki.battle.mqtt.MqttService;
 import de.adesso.anki.battle.util.Position;
+import de.adesso.anki.battle.world.bodies.Rocket;
+import de.adesso.anki.battle.world.bodies.Vehicle;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'de.adesso.anki.battle.world.bodies.Vehicle'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)